index.html

```diff
@@ -15,33 +15,66 @@ <h2>Gaming ID Card</h2>
     <section class="section-main">
       <nav>
         <ul>
-          <li class="li-playstation"><img class="image-playstation" src="https://png2.cleanpng.com/sh/f4dec2680c6b5dbde9ffd28352d75336/L0KzQYm3U8IxN6F6j5H0aYP2gLBuTfJzaZ9pRd54Z3Awc7r5gBxmNaFxeet8dHH3ebF1TcViamFpSKQDNkKzdbWCTsk1QGg4SqUCMUW1QYa6V8A3PGU1Sak3cH7q/kisspng-brand-logo-circle-playstation-5ab0d028620ed9.9487323715215370644017.png" alt="playstation logo"> Playstation</li>
-          <input type="text">
-          
+          <li class="li-playstation">
+            <img
+              class="image-playstation"
+              src="https://png2.cleanpng.com/sh/f4dec2680c6b5dbde9ffd28352d75336/L0KzQYm3U8IxN6F6j5H0aYP2gLBuTfJzaZ9pRd54Z3Awc7r5gBxmNaFxeet8dHH3ebF1TcViamFpSKQDNkKzdbWCTsk1QGg4SqUCMUW1QYa6V8A3PGU1Sak3cH7q/kisspng-brand-logo-circle-playstation-5ab0d028620ed9.9487323715215370644017.png"
+              alt="playstation logo"
+            />
+            Playstation
+          </li>
+          <!-- Playstation Input Start-->
+          <input id="playstation-input" type="text" />
+          <button onclick="playstationFunction()" id="playstation-write">
```

Comment:
You mix onclick and addEventListener now. It is great that you used this, but try to use addEventListener for the learning and code quality.